docs/development.md

```diff
@@ -52,13 +54,71 @@ Lastly, spin up the box:
 vagrant up centos7-devel
 ```
 
-The box can now be accessed via ssh and the Rails server started directly (this assumes you are connecting as the default `vagrant` user):
+If you get an error about the host CPU not providing the 'svm' feature but you most assuredly do have an Intel CPU and virtualization is most definitely turned on, add this to your Vagrantfile.
 
-    vagrant ssh <deployment>
-    cd /home/vagrant/foreman
-    sudo service iptables stop
-    bin/rails s -b 0.0.0.0
+```
+Vagrant.configure("2") do |config|
+  config.vm.provider :libvirt do |domain|
+    domain.cpu_mode = "host-passthrough"
+  end
+end
+```
+
+Try to bring it up again with `vagrant up centos7-devel` after making that change.
+
+If you get an error related to your user not having privileges to execute libvirt commands, create a polkit rule similar to the following:
+
+```
+polkit.addRule(function(action, subject) {
+    if (action.id == "org.libvirt.unix.manage" &&
+        subject.isInGroup("wheel")) {
+            return polkit.Result.YES;
+    }
+});
+```
+
+This rule will allow any user in the `wheel` group to perform libvirt commands. The file needs to reside in `/etc/polkit-1/rules.d`, named something like `49-org.libvirt.unix.manage.rules`. Now try to bring it up again with `vagrant up centos7-devel`.
+
+You may experience some failures during the provisioning process. You may also find that rerunning `vagrant provision centos7-devel` will eventually result in a successful installation. If not, check error messages and logs for hints and/or seek help from the mailing list or IRC.
 
+Once successfulling installed, access the box via ssh: `vagrant ssh centos7-devel`.
+
+Before we can run the app, we need to install a couple things by hand to make webpack work.
+
+Install `rh-nodejs8-npm` and `http-parser-devel`:
+
+```
+yum install rh-nodejs8-npm http-parser-devel
+```
+
+Enable the nodejs installation:
+
+```
+source /opt/rh/rh-nodejs8/enable
+```
```

Comment:
From my experience, the SCL is not needed to perform npm actions, what's in EPEL is good enough so I'd prefer if we left this out of the change.